addons/web/static/src/js/views/list/list_view.js

```diff
@@ -94,6 +99,20 @@ var ListView = BasicView.extend({
         var innerView = this.fields[node.attrs.name].views.groupby;
         this.groupbys[node.attrs.name] = this._processFieldsView(innerView, 'groupby');
     },
+    /**
+     * Extracts action buttons definitions from the <header>
+     * node of the list view definition
```

Comment:
Which pattern do you follow to determine when to do line breaks in docstrings? I personnaly go for 80 char, so it would be:
```js
    /**
     * Extracts action buttons definitions from the <header> node of the list
     * view definition
     */
```